apps/delphi_rest_api.cpp

```diff
@@ -165,18 +128,11 @@ class Experiment {
                     double lower_limit = time_step[lower_rank];
                     double upper_limit = time_step[upper_rank];
 
-                    unordered_map<string, double> value_dict = {
-                        {"timestamp", timesteps_nparr[i]},
-                        {"value", median_value}};
-
-                    data_dict["values"].push_back(value_dict);
-                    value_dict["value"] = lower_limit;
-                    data_dict["confidenceInterval"]["lower"].push_back(
-                        value_dict);
+		    json tseries;
+		    tseries["timestamp"] = timesteps_nparr[i];
+		    tseries["values"] = vector<double>{lower_limit, median_value,upper_limit};
```

Comment:
Here we have to send the raw prediction results instead of these. So we just have to send the time_steps
vector.

Looking back at the specification I see the confusion. They have only mentioned that we have to send an array but omitted what the contents of the array. I know this only because I was present at the discussion where this decision was made public by Uncharted.

Reply:
OK, the entire time_step vector is now sent for each timestamp.